tests/acceptance-bdd/test_literature_suggestion_form.py

```diff
@@ -0,0 +1,392 @@
+    )
+    Arsenic().execute_script(
+        """document.evaluate(
+            "//div[@id='webdeposit_form_accordion']/div[3]/div[9]/div[1]",
+            document,
+            null,
+            XPathResult.FIRST_ORDERED_NODE_TYPE,
+            null
+        ).singleNodeValue.click()"""
+    )
+    Arsenic().show_title_bar()
+
+
+@when(parsers.cfparse('I select the value {input_select} in the select box with id {field_id}'))
+def select_generic_value(input_select, field_id):
+    Select(Arsenic().find_element_by_id(field_id)).select_by_value(input_select)
+
+
+@when(parsers.cfparse('I select the value {input_select} in the subject dropdown list'))
+def select_generic_value_dropdown(input_select):
+    Arsenic().find_element_by_xpath('(//button[@type="button"])[8]').click()
+    Arsenic().find_element_by_css_selector('input[type=\"checkbox\"]').click()
+    Arsenic().find_element_by_xpath(
+        '//input[@value="' + input_select + '"]'
+    ).click()
+    Arsenic().find_element_by_xpath('(//button[@type="button"])[8]').click()
+
+
+@then(parsers.cfparse('I should see the {field} error message'))
+def is_there_a_message_error(field):
+    assert globals()['_is_' + field + '_error_message_present']()
+
+
+@then(parsers.cfparse('I should not see the {field} error message'))
+def is_not_there_a_message_error(field):
+    assert not globals()['_is_' + field + '_error_message_present']()
+
+
+@when(parsers.cfparse('I insert <input> in the input box with id {field_id}'))
+@when(parsers.cfparse('I insert {input} in the input box with id {field_id}'))
+def write_in_a_generic_field(input, field_id):
+    Arsenic().write_in_generic_field(field_id, input)
+
+
+@when(
+    parsers.cfparse(
+        'I insert {input_autocompletion} in the autocomplete input box with id {autocompletion_field}'
+    )
+)
+def write_in_a_generic_autocompletion_field(input_autocompletion, autocompletion_field):
+    Arsenic().write_in_autocomplete_field(autocompletion_field, input_autocompletion)
+
+
+@then(parsers.cfparse('I should see {expected_data} in the input box with id {field_id}'))
+def check_generic_field(expected_data, field_id):
+    field = Arsenic().find_element_by_id(field_id)
+    assert expected_data == field.get_attribute('value')
+
+
+@then(parsers.cfparse('I should see {value_record_entry} in the record entry'))
+def check_record_detail(value_record_entry):
+    record = WebDriverWait(Arsenic(), 10).until(
+        EC.visibility_of_element_located(
+            (
+                By.XPATH,
+                '//div[@class="row hp-item ng-scope"][1]'
+            )
+        )
+    ).text
+    assert value_record_entry in record
+
+
+@then(parsers.cfparse('I should see {value_record_detail} in the record detail'))
+def check_record_detail(value_record_detail):
+    record = WebDriverWait(Arsenic(), 10).until(
+        EC.visibility_of_element_located(
+            (
+                By.XPATH,
+                '(//div[@class="ng-scope"])[2]'
```

Comment:
Same here and the selectors below, and `id` in the html will make it more readable.